### ISSUE TITLE
Device orientation problem

### ISSUE THREAD

**Issue by Sheimy** [shubham0204/FaceRecognition_With_FaceNet_Android#26]
when i run the app on tablet with landscap orientation the recognation is alaways unknown

**Comment by shubham0204** [comment]
@Sheimy I will look into the issue and get back here.

**Comment by omikesharwani22** [comment]
@Sheimy yes, it is not working when I move my mobile portrait to landscape.

**Comment by shubham0204** [comment]
@omikesharwani22 I am looking into the issue and will get back to you within a few days.
